projects/libvpx/vpx_dec_fuzzer_vp8-afl++.toml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+exit-on-time = 7200
+
+[sydr]
+args = "-l trace -s 180"
+target = "/libvpx/sydr/vpx_dec_fuzzer_vp8 @@"
+jobs = 1
+
+[[aflplusplus]]
```

Comment:
'[aflplusplus]' if there are no any addition subtables for afl

Reply:
ty